### PR TITLE
[Doc] Add readme to mxnet gat

### PR DESCRIPTION
**Pull request by sufeidechabei**

### CODE REVIEW

**Comment by jermainewang** [comment]
Thanks. Could you also add a reminder in the README.md of the mxnet example folders about how to switch to mxnet backend?

**Comment by ylfdq1118** [comment]
@sufeidechabei Also, can you follow the readme files of other example models? 

You can remove the paper abstract from readme. Instead, you should include paper link, author implementation link, and how to run your code (including how to use mxnet backend).

You can also include some performance results (training accuracy of DGL vs author).

**Comment by sufeidechabei** [comment]
Okay.

**Comment by jermainewang** [comment]
@sufeidechabei any updates?

**Comment by sufeidechabei** [comment]
GAT (pytorch gat and mxnet gat) have bugs. I'm fixing them now.

**Comment by jermainewang** [comment]
Let's fix one thing at a time. You can fix the readme first. And use another PR for the bugfix.

**Comment by sufeidechabei** [comment]
I have changed the readme.

File: examples/mxnet/README.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
 # Model Examples using DGL (w/ MXNet backend)
+```
+use DGLBACKEND=mxnet to convert DGLBACKEND to mxnet.
+```
```
Comment (ylfdq1118) [inline examples/mxnet/README.md]: Code block should only have code in it. Change to:
use `DGLBACKEND=mxnet` to use MXNet as DGL's backend

File: examples/mxnet/gat/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Graph Attention Networks (GAT)
+============
+
+- Paper link: [https://arxiv.org/abs/1710.10903](https://arxiv.org/abs/1710.10903)
+- Author's code repo:
+  [https://github.com/PetarV-/GAT](https://github.com/PetarV-/GAT).
+
+Note that the original code is implemented with Tensorflow for the paper.
+
+[[Paper]](https://arxiv.org/abs/1710.10903) [[Original Implementation]](https://github.com/PetarV-/GAT)
```
Comment (ylfdq1118) [inline examples/mxnet/gat/README.md]: You can remove line 10 since it's a duplicate of the content above.

File: examples/mxnet/README.md (hunk 1 above)
Comment (sufeidechabei) [reply examples/mxnet/README.md]: I have fixed that.

Comment (yzh119) [reply examples/mxnet/README.md]: the DGLBACKEND=mxnet should be wrapped with a pair of backquote, if you are not familiar with that, check the following:
```
Use `DGLBACKEND=mxnet` to use MXNet as DGL's backend.
```

Comment (sufeidechabei) [reply examples/mxnet/README.md]: Done
